airflow/config_templates/default_airflow.cfg

```diff
@@ -678,6 +678,11 @@ in_cluster = True
 #   https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#affinity-v1-core
 affinity =
 
+# Lifecycle configuration as a single line formatted JSON object.
+# See the lifecycle documentation:
+#   https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#lifecycle-v1-core
+life_cycle =
```

Comment:
I wouldn't want to have this one step in json when all other steps are not json objects. Also why not just have this be an option you can place into the operator itself?

Reply:
Just looked a bit further down. Can you split doing this for the pod operator and k8s executor into separate tickets/PRs?